.travis/validate-package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Script to validate our packed package
+
+npm pack
+tar -xzf solid-community-server-*.tgz
+pushd package
+
+# Check if our server can start at a given port
+node bin/server.js -p 8888 &
```

Comment:
I wonder if we can do this with an `npx solid-community-server` to also test the bin name (but might not be the case). Or somehow even an `node -e` to get the field out of the JSON. Not crucial though.

Reply:
`npx` seems to work indeed!